aws/logs_monitoring/lambda_function.py

```diff
@@ -461,6 +481,17 @@ def generate_metadata(context):
     return metadata
 
 
+def extract_traces(event):
```

Comment:
Is it `trace` or `traces`? I believe one message can only contain spans from the same trace?